node/tools/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY: run-node node-configs clean
+IP?=127.0.0.1:8000
```

Comment:
is this a good default? This port has some canonical uses https://www.speedguide.net/port.php?port=8000 , so perhaps something more random would be preferred.

Reply:
Updated here [9fe5437](https://github.com/matter-labs/era-consensus/pull/55/commits/9fe5437f3bda51f8010a2264f4aa912ca4c1229c)!